src/Templates/src/templates/maui-blazor/Platforms/MacCatalyst/Entitlements.Debug.plist

```diff
@@ -1,10 +0,0 @@
-﻿<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-<plist version="1.0">
-    <dict>
-        <!-- Enable this value to use browser developer tools while debugging.-->
-        <!-- See https://aka.ms/blazor-hybrid-developer-tools -->
-        <key>com.apple.security.get-task-allow</key>
-        <true/>
```

Comment:
Is this no longer needed?

Reply:
Correct, is added via the iOS sdk: https://github.com/xamarin/xamarin-macios/pull/18669